From 0de4eb48266afb2572440ea6fcc6bb9260cb0371 Mon Sep 17 00:00:00 2001
From: John Ma <jma@phytec.com>
Date: Tue, 11 Nov 2025 18:30:40 -0800
Subject: [PATCH 2/2] arch: arm: dts: k3-am67-phycore-som-lpddr4-8gb: Update
DDR timings

Update the DDR timings for the phyCORE-AM67x to use a 8GB module.
Timings were generated using J722S DDRSS Register Configuration Tool
v0.12.0.
Rename dtsi filename to 8gb.

Signed-off-by: John Ma <jma@phytec.com>
---
 ...si => k3-am67-phycore-som-lpddr4-8gb.dtsi} | 434 +++++++++---------
 arch/arm/dts/k3-am6754-r5-phycore-som-8gb.dts |   2 +-
 2 files changed, 218 insertions(+), 218 deletions(-)
 rename arch/arm/dts/{k3-am67-phycore-som-lpddr4-4gb.dtsi => k3-am67-phycore-som-lpddr4-8gb.dtsi} (92%)

diff --git a/arch/arm/dts/k3-am67-phycore-som-lpddr4-4gb.dtsi b/arch/arm/dts/k3-am67-phycore-som-lpddr4-8gb.dtsi
similarity index 92%
rename from arch/arm/dts/k3-am67-phycore-som-lpddr4-4gb.dtsi
rename to arch/arm/dts/k3-am67-phycore-som-lpddr4-8gb.dtsi
index a06aacaf674..dbaacd5621f 100644
--- a/arch/arm/dts/k3-am67-phycore-som-lpddr4-4gb.dtsi
+++ b/arch/arm/dts/k3-am67-phycore-som-lpddr4-8gb.dtsi
@@ -2,16 +2,16 @@
 /*
  * Copyright (C) 2023 Texas Instruments Incorporated - http://www.ti.com/
  * This file was generated with the following tool revisions:
- *     - SysConfig: Revision 1.24.2+4234
+ *     - SysConfig: Revision 1.25.0+4268
  *     - Jacinto7_DDRSS_RegConfigTool: Revision 0.12.0
- * This file was generated on Thu Aug 28 2025 08:32:48 GMT-0700 (Pacific Daylight Time)
+ * This file was generated on Fri Oct 31 2025 14:50:48 GMT-0700 (Pacific Daylight Time)
 */
 
 
-#define DDRSS_PLL_FHS_CNT 3
+#define DDRSS_PLL_FHS_CNT 5
 #define DDRSS_PLL_FREQUENCY_0 25000000
-#define DDRSS_PLL_FREQUENCY_1 933000000
-#define DDRSS_PLL_FREQUENCY_2 933000000
+#define DDRSS_PLL_FREQUENCY_1 1066500000
+#define DDRSS_PLL_FREQUENCY_2 1066500000
 
 #define DDRSS_CTL_0_DATA 0x00000B00
 #define DDRSS_CTL_1_DATA 0x00000000
@@ -24,14 +24,14 @@
 #define DDRSS_CTL_8_DATA 0x000186A0
 #define DDRSS_CTL_9_DATA 0x00000005
 #define DDRSS_CTL_10_DATA 0x00000064
-#define DDRSS_CTL_11_DATA 0x0005B18F
-#define DDRSS_CTL_12_DATA 0x0038EF90
+#define DDRSS_CTL_11_DATA 0x000681C8
+#define DDRSS_CTL_12_DATA 0x004111C9
 #define DDRSS_CTL_13_DATA 0x00000005
-#define DDRSS_CTL_14_DATA 0x00000E94
-#define DDRSS_CTL_15_DATA 0x0005B18F
-#define DDRSS_CTL_16_DATA 0x0038EF90
+#define DDRSS_CTL_14_DATA 0x000010A9
+#define DDRSS_CTL_15_DATA 0x000681C8
+#define DDRSS_CTL_16_DATA 0x004111C9
 #define DDRSS_CTL_17_DATA 0x00000005
-#define DDRSS_CTL_18_DATA 0x00000E94
+#define DDRSS_CTL_18_DATA 0x000010A9
 #define DDRSS_CTL_19_DATA 0x01010100
 #define DDRSS_CTL_20_DATA 0x01010100
 #define DDRSS_CTL_21_DATA 0x01000110
@@ -48,7 +48,7 @@
 #define DDRSS_CTL_32_DATA 0x00000000
 #define DDRSS_CTL_33_DATA 0x00000000
 #define DDRSS_CTL_34_DATA 0x08000010
-#define DDRSS_CTL_35_DATA 0x00004B4B
+#define DDRSS_CTL_35_DATA 0x00005656
 #define DDRSS_CTL_36_DATA 0x00000000
 #define DDRSS_CTL_37_DATA 0x00000000
 #define DDRSS_CTL_38_DATA 0x00000000
@@ -63,62 +63,62 @@
 #define DDRSS_CTL_47_DATA 0x00000800
 #define DDRSS_CTL_48_DATA 0x09090004
 #define DDRSS_CTL_49_DATA 0x00000204
-#define DDRSS_CTL_50_DATA 0x007A0012
-#define DDRSS_CTL_51_DATA 0x09140054
-#define DDRSS_CTL_52_DATA 0x00003A26
-#define DDRSS_CTL_53_DATA 0x007A0012
-#define DDRSS_CTL_54_DATA 0x09140054
-#define DDRSS_CTL_55_DATA 0x09003A26
+#define DDRSS_CTL_50_DATA 0x008B0015
+#define DDRSS_CTL_51_DATA 0x09170060
+#define DDRSS_CTL_52_DATA 0x0000422B
+#define DDRSS_CTL_53_DATA 0x008B0015
+#define DDRSS_CTL_54_DATA 0x09170060
+#define DDRSS_CTL_55_DATA 0x0900422B
 #define DDRSS_CTL_56_DATA 0x000A0A09
 #define DDRSS_CTL_57_DATA 0x0400036D
-#define DDRSS_CTL_58_DATA 0x090F2005
-#define DDRSS_CTL_59_DATA 0x00001B13
-#define DDRSS_CTL_60_DATA 0x0E007FE6
-#define DDRSS_CTL_61_DATA 0x090F200F
-#define DDRSS_CTL_62_DATA 0x00001B13
-#define DDRSS_CTL_63_DATA 0x0E007FE6
-#define DDRSS_CTL_64_DATA 0x0304200F
+#define DDRSS_CTL_58_DATA 0x09112005
+#define DDRSS_CTL_59_DATA 0x00001E16
+#define DDRSS_CTL_60_DATA 0x1000922C
+#define DDRSS_CTL_61_DATA 0x09112011
+#define DDRSS_CTL_62_DATA 0x00001E16
+#define DDRSS_CTL_63_DATA 0x1000922C
+#define DDRSS_CTL_64_DATA 0x03042011
 #define DDRSS_CTL_65_DATA 0x04050002
-#define DDRSS_CTL_66_DATA 0x24262426
+#define DDRSS_CTL_66_DATA 0x292C292C
 #define DDRSS_CTL_67_DATA 0x01010008
-#define DDRSS_CTL_68_DATA 0x044A4A08
-#define DDRSS_CTL_69_DATA 0x042B2B04
-#define DDRSS_CTL_70_DATA 0x00002B2B
+#define DDRSS_CTL_68_DATA 0x04545408
+#define DDRSS_CTL_69_DATA 0x04313104
+#define DDRSS_CTL_70_DATA 0x00003131
 #define DDRSS_CTL_71_DATA 0x00000101
 #define DDRSS_CTL_72_DATA 0x00000000
 #define DDRSS_CTL_73_DATA 0x01000000
 #define DDRSS_CTL_74_DATA 0x00130803
 #define DDRSS_CTL_75_DATA 0x0000005E
-#define DDRSS_CTL_76_DATA 0x000002C5
-#define DDRSS_CTL_77_DATA 0x00000E2E
-#define DDRSS_CTL_78_DATA 0x000002C5
-#define DDRSS_CTL_79_DATA 0x00000E2E
+#define DDRSS_CTL_76_DATA 0x0000032B
+#define DDRSS_CTL_77_DATA 0x00001035
+#define DDRSS_CTL_78_DATA 0x0000032B
+#define DDRSS_CTL_79_DATA 0x00001035
 #define DDRSS_CTL_80_DATA 0x00000005
 #define DDRSS_CTL_81_DATA 0x0000000A
 #define DDRSS_CTL_82_DATA 0x00000010
-#define DDRSS_CTL_83_DATA 0x00000163
-#define DDRSS_CTL_84_DATA 0x00000386
-#define DDRSS_CTL_85_DATA 0x00000163
-#define DDRSS_CTL_86_DATA 0x00000386
+#define DDRSS_CTL_83_DATA 0x00000196
+#define DDRSS_CTL_84_DATA 0x00000408
+#define DDRSS_CTL_85_DATA 0x00000196
+#define DDRSS_CTL_86_DATA 0x00000408
 #define DDRSS_CTL_87_DATA 0x03004000
 #define DDRSS_CTL_88_DATA 0x00001201
-#define DDRSS_CTL_89_DATA 0x000E0005
-#define DDRSS_CTL_90_DATA 0x2908000E
-#define DDRSS_CTL_91_DATA 0x0A050529
-#define DDRSS_CTL_92_DATA 0x1B0E0A03
-#define DDRSS_CTL_93_DATA 0x1B0E0A04
+#define DDRSS_CTL_89_DATA 0x00100005
+#define DDRSS_CTL_90_DATA 0x2F080010
+#define DDRSS_CTL_91_DATA 0x0A05052F
+#define DDRSS_CTL_92_DATA 0x1E100B03
+#define DDRSS_CTL_93_DATA 0x1E100B04
 #define DDRSS_CTL_94_DATA 0x04010104
 #define DDRSS_CTL_95_DATA 0x00010401
 #define DDRSS_CTL_96_DATA 0x00140014
-#define DDRSS_CTL_97_DATA 0x02D302D3
-#define DDRSS_CTL_98_DATA 0x02D302D3
+#define DDRSS_CTL_97_DATA 0x033B033B
+#define DDRSS_CTL_98_DATA 0x033B033B
 #define DDRSS_CTL_99_DATA 0x00000000
 #define DDRSS_CTL_100_DATA 0x03030000
 #define DDRSS_CTL_101_DATA 0x05050501
-#define DDRSS_CTL_102_DATA 0x04041C04
-#define DDRSS_CTL_103_DATA 0x0E0A0E0A
-#define DDRSS_CTL_104_DATA 0x0A04041C
-#define DDRSS_CTL_105_DATA 0x030E0A0E
+#define DDRSS_CTL_102_DATA 0x04042004
+#define DDRSS_CTL_103_DATA 0x100B100B
+#define DDRSS_CTL_104_DATA 0x0B040420
+#define DDRSS_CTL_105_DATA 0x03100B10
 #define DDRSS_CTL_106_DATA 0x00000404
 #define DDRSS_CTL_107_DATA 0x00000301
 #define DDRSS_CTL_108_DATA 0x00000001
@@ -139,20 +139,20 @@
 #define DDRSS_CTL_123_DATA 0x00001640
 #define DDRSS_CTL_124_DATA 0x00000000
 #define DDRSS_CTL_125_DATA 0x0000026F
-#define DDRSS_CTL_126_DATA 0x00038B80
-#define DDRSS_CTL_127_DATA 0x00038B80
-#define DDRSS_CTL_128_DATA 0x00038B80
-#define DDRSS_CTL_129_DATA 0x00038B80
-#define DDRSS_CTL_130_DATA 0x00038B80
+#define DDRSS_CTL_126_DATA 0x00040D40
+#define DDRSS_CTL_127_DATA 0x00040D40
+#define DDRSS_CTL_128_DATA 0x00040D40
+#define DDRSS_CTL_129_DATA 0x00040D40
+#define DDRSS_CTL_130_DATA 0x00040D40
 #define DDRSS_CTL_131_DATA 0x00000000
-#define DDRSS_CTL_132_DATA 0x00006342
-#define DDRSS_CTL_133_DATA 0x00038B80
-#define DDRSS_CTL_134_DATA 0x00038B80
-#define DDRSS_CTL_135_DATA 0x00038B80
-#define DDRSS_CTL_136_DATA 0x00038B80
-#define DDRSS_CTL_137_DATA 0x00038B80
+#define DDRSS_CTL_132_DATA 0x00007173
+#define DDRSS_CTL_133_DATA 0x00040D40
+#define DDRSS_CTL_134_DATA 0x00040D40
+#define DDRSS_CTL_135_DATA 0x00040D40
+#define DDRSS_CTL_136_DATA 0x00040D40
+#define DDRSS_CTL_137_DATA 0x00040D40
 #define DDRSS_CTL_138_DATA 0x00000000
-#define DDRSS_CTL_139_DATA 0x00006342
+#define DDRSS_CTL_139_DATA 0x00007173
 #define DDRSS_CTL_140_DATA 0x00000000
 #define DDRSS_CTL_141_DATA 0x00000000
 #define DDRSS_CTL_142_DATA 0x00000000
@@ -172,7 +172,7 @@
 #define DDRSS_CTL_156_DATA 0x00000000
 #define DDRSS_CTL_157_DATA 0x00000000
 #define DDRSS_CTL_158_DATA 0x03050000
-#define DDRSS_CTL_159_DATA 0x040A040A
+#define DDRSS_CTL_159_DATA 0x040B040B
 #define DDRSS_CTL_160_DATA 0x00000000
 #define DDRSS_CTL_161_DATA 0x07010A09
 #define DDRSS_CTL_162_DATA 0x000E0A09
@@ -211,12 +211,12 @@
 #define DDRSS_CTL_195_DATA 0x0005000A
 #define DDRSS_CTL_196_DATA 0x0404000D
 #define DDRSS_CTL_197_DATA 0x0000000D
-#define DDRSS_CTL_198_DATA 0x00BB0176
-#define DDRSS_CTL_199_DATA 0x0E0E01D3
-#define DDRSS_CTL_200_DATA 0x000001D3
-#define DDRSS_CTL_201_DATA 0x00BB0176
-#define DDRSS_CTL_202_DATA 0x0E0E01D3
-#define DDRSS_CTL_203_DATA 0x000001D3
+#define DDRSS_CTL_198_DATA 0x00D601AB
+#define DDRSS_CTL_199_DATA 0x10100216
+#define DDRSS_CTL_200_DATA 0x00000216
+#define DDRSS_CTL_201_DATA 0x00D601AB
+#define DDRSS_CTL_202_DATA 0x10100216
+#define DDRSS_CTL_203_DATA 0x00000216
 #define DDRSS_CTL_204_DATA 0x00000000
 #define DDRSS_CTL_205_DATA 0x00000000
 #define DDRSS_CTL_206_DATA 0x00000000
@@ -328,25 +328,25 @@
 #define DDRSS_CTL_312_DATA 0x00400100
 #define DDRSS_CTL_313_DATA 0x00080032
 #define DDRSS_CTL_314_DATA 0x01000200
-#define DDRSS_CTL_315_DATA 0x074A0040
-#define DDRSS_CTL_316_DATA 0x00020038
+#define DDRSS_CTL_315_DATA 0x08550040
+#define DDRSS_CTL_316_DATA 0x00020040
 #define DDRSS_CTL_317_DATA 0x00400100
-#define DDRSS_CTL_318_DATA 0x0038074A
+#define DDRSS_CTL_318_DATA 0x00400855
 #define DDRSS_CTL_319_DATA 0x00030000
-#define DDRSS_CTL_320_DATA 0x005E005E
+#define DDRSS_CTL_320_DATA 0x006B006B
 #define DDRSS_CTL_321_DATA 0x00000100
 #define DDRSS_CTL_322_DATA 0x01010000
-#define DDRSS_CTL_323_DATA 0x00000000
-#define DDRSS_CTL_324_DATA 0x3FFF0000
+#define DDRSS_CTL_323_DATA 0x00000101
+#define DDRSS_CTL_324_DATA 0x1FFF0000
 #define DDRSS_CTL_325_DATA 0x000FFF00
-#define DDRSS_CTL_326_DATA 0xFFFFFFFF
-#define DDRSS_CTL_327_DATA 0x00FFFF00
-#define DDRSS_CTL_328_DATA 0x0B000000
+#define DDRSS_CTL_326_DATA 0x3FFF2000
+#define DDRSS_CTL_327_DATA 0x000FFF00
+#define DDRSS_CTL_328_DATA 0x0B000001
 #define DDRSS_CTL_329_DATA 0x0001FFFF
 #define DDRSS_CTL_330_DATA 0x01010101
 #define DDRSS_CTL_331_DATA 0x01010101
 #define DDRSS_CTL_332_DATA 0x00000118
-#define DDRSS_CTL_333_DATA 0x00000C01
+#define DDRSS_CTL_333_DATA 0x00000C03
 #define DDRSS_CTL_334_DATA 0x00040100
 #define DDRSS_CTL_335_DATA 0x00040100
 #define DDRSS_CTL_336_DATA 0x00000000
@@ -399,16 +399,16 @@
 #define DDRSS_CTL_383_DATA 0x01000101
 #define DDRSS_CTL_384_DATA 0x01010001
 #define DDRSS_CTL_385_DATA 0x00010101
-#define DDRSS_CTL_386_DATA 0x01090903
+#define DDRSS_CTL_386_DATA 0x01080803
 #define DDRSS_CTL_387_DATA 0x05020201
-#define DDRSS_CTL_388_DATA 0x0E081B1B
-#define DDRSS_CTL_389_DATA 0x0009040E
-#define DDRSS_CTL_390_DATA 0x0B0D040F
-#define DDRSS_CTL_391_DATA 0x0B0D0406
-#define DDRSS_CTL_392_DATA 0x0D0D0906
+#define DDRSS_CTL_388_DATA 0x10081B1B
+#define DDRSS_CTL_389_DATA 0x00090410
+#define DDRSS_CTL_390_DATA 0x0B0C040F
+#define DDRSS_CTL_391_DATA 0x0B0C0406
+#define DDRSS_CTL_392_DATA 0x0C0C0906
 #define DDRSS_CTL_393_DATA 0x01000000
-#define DDRSS_CTL_394_DATA 0x07030701
-#define DDRSS_CTL_395_DATA 0x04000103
+#define DDRSS_CTL_394_DATA 0x08040801
+#define DDRSS_CTL_395_DATA 0x04000104
 #define DDRSS_CTL_396_DATA 0x1B000004
 #define DDRSS_CTL_397_DATA 0x000000B2
 #define DDRSS_CTL_398_DATA 0x00000200
@@ -418,24 +418,24 @@
 #define DDRSS_CTL_402_DATA 0x0000034E
 #define DDRSS_CTL_403_DATA 0x000006F4
 #define DDRSS_CTL_404_DATA 0x03000202
-#define DDRSS_CTL_405_DATA 0x37200201
-#define DDRSS_CTL_406_DATA 0x00001C5C
+#define DDRSS_CTL_405_DATA 0x3A200201
+#define DDRSS_CTL_406_DATA 0x0000206A
 #define DDRSS_CTL_407_DATA 0x00000200
 #define DDRSS_CTL_408_DATA 0x00000200
 #define DDRSS_CTL_409_DATA 0x00000200
 #define DDRSS_CTL_410_DATA 0x00000200
-#define DDRSS_CTL_411_DATA 0x00007F9E
-#define DDRSS_CTL_412_DATA 0x00011B98
-#define DDRSS_CTL_413_DATA 0x111A0402
-#define DDRSS_CTL_414_DATA 0x37200C09
-#define DDRSS_CTL_415_DATA 0x00001C5C
+#define DDRSS_CTL_411_DATA 0x000091DD
+#define DDRSS_CTL_412_DATA 0x00014424
+#define DDRSS_CTL_413_DATA 0x101A0402
+#define DDRSS_CTL_414_DATA 0x3A200C09
+#define DDRSS_CTL_415_DATA 0x0000206A
 #define DDRSS_CTL_416_DATA 0x00000200
 #define DDRSS_CTL_417_DATA 0x00000200
 #define DDRSS_CTL_418_DATA 0x00000200
 #define DDRSS_CTL_419_DATA 0x00000200
-#define DDRSS_CTL_420_DATA 0x00007F9E
-#define DDRSS_CTL_421_DATA 0x00011B98
-#define DDRSS_CTL_422_DATA 0x111A0402
+#define DDRSS_CTL_420_DATA 0x000091DD
+#define DDRSS_CTL_421_DATA 0x00014424
+#define DDRSS_CTL_422_DATA 0x101A0402
 #define DDRSS_CTL_423_DATA 0x00200C09
 #define DDRSS_CTL_424_DATA 0x00000000
 #define DDRSS_CTL_425_DATA 0x02000A00
@@ -461,7 +461,7 @@
 #define DDRSS_PI_10_DATA 0x00000000
 #define DDRSS_PI_11_DATA 0x00000002
 #define DDRSS_PI_12_DATA 0x00000005
-#define DDRSS_PI_13_DATA 0x00050001
+#define DDRSS_PI_13_DATA 0x000F0001
 #define DDRSS_PI_14_DATA 0x08000000
 #define DDRSS_PI_15_DATA 0x00010300
 #define DDRSS_PI_16_DATA 0x00000005
@@ -477,7 +477,7 @@
 #define DDRSS_PI_26_DATA 0x01010000
 #define DDRSS_PI_27_DATA 0x0A000100
 #define DDRSS_PI_28_DATA 0x00000028
-#define DDRSS_PI_29_DATA 0x05000000
+#define DDRSS_PI_29_DATA 0x0F000000
 #define DDRSS_PI_30_DATA 0x00320000
 #define DDRSS_PI_31_DATA 0x00000000
 #define DDRSS_PI_32_DATA 0x00000000
@@ -496,8 +496,8 @@
 #define DDRSS_PI_45_DATA 0x000000B5
 #define DDRSS_PI_46_DATA 0x00000000
 #define DDRSS_PI_47_DATA 0x00000000
-#define DDRSS_PI_48_DATA 0x00050500
-#define DDRSS_PI_49_DATA 0x0000001A
+#define DDRSS_PI_48_DATA 0x000F0F00
+#define DDRSS_PI_49_DATA 0x0000001B
 #define DDRSS_PI_50_DATA 0x000007D0
 #define DDRSS_PI_51_DATA 0x00000300
 #define DDRSS_PI_52_DATA 0x00000000
@@ -507,7 +507,7 @@
 #define DDRSS_PI_56_DATA 0x01000000
 #define DDRSS_PI_57_DATA 0x03000000
 #define DDRSS_PI_58_DATA 0x00000000
-#define DDRSS_PI_59_DATA 0x00001705
+#define DDRSS_PI_59_DATA 0x0000170F
 #define DDRSS_PI_60_DATA 0x00000000
 #define DDRSS_PI_61_DATA 0x00000000
 #define DDRSS_PI_62_DATA 0x00000000
@@ -517,7 +517,7 @@
 #define DDRSS_PI_66_DATA 0x05000404
 #define DDRSS_PI_67_DATA 0x00010001
 #define DDRSS_PI_68_DATA 0x0001000E
-#define DDRSS_PI_69_DATA 0x01010500
+#define DDRSS_PI_69_DATA 0x01010F00
 #define DDRSS_PI_70_DATA 0x00010000
 #define DDRSS_PI_71_DATA 0x00000034
 #define DDRSS_PI_72_DATA 0x00000000
@@ -528,7 +528,7 @@
 #define DDRSS_PI_77_DATA 0x00000000
 #define DDRSS_PI_78_DATA 0x00000000
 #define DDRSS_PI_79_DATA 0x01000000
-#define DDRSS_PI_80_DATA 0x00010001
+#define DDRSS_PI_80_DATA 0x01010001
 #define DDRSS_PI_81_DATA 0x02000008
 #define DDRSS_PI_82_DATA 0x01000200
 #define DDRSS_PI_83_DATA 0x00000100
@@ -625,11 +625,11 @@
 #define DDRSS_PI_174_DATA 0x05010000
 #define DDRSS_PI_175_DATA 0x00000001
 #define DDRSS_PI_176_DATA 0x00080000
-#define DDRSS_PI_177_DATA 0x01180118
-#define DDRSS_PI_178_DATA 0x00262601
+#define DDRSS_PI_177_DATA 0x01400140
+#define DDRSS_PI_178_DATA 0x002B2B01
 #define DDRSS_PI_179_DATA 0x00000034
-#define DDRSS_PI_180_DATA 0x0000005E
-#define DDRSS_PI_181_DATA 0x0002005E
+#define DDRSS_PI_180_DATA 0x00000060
+#define DDRSS_PI_181_DATA 0x00020060
 #define DDRSS_PI_182_DATA 0x02000200
 #define DDRSS_PI_183_DATA 0x00000004
 #define DDRSS_PI_184_DATA 0x0000100C
@@ -637,110 +637,110 @@
 #define DDRSS_PI_186_DATA 0x00400000
 #define DDRSS_PI_187_DATA 0x00000013
 #define DDRSS_PI_188_DATA 0x0000005E
-#define DDRSS_PI_189_DATA 0x000002C5
-#define DDRSS_PI_190_DATA 0x00000E2E
-#define DDRSS_PI_191_DATA 0x000002C5
-#define DDRSS_PI_192_DATA 0x04000E2E
+#define DDRSS_PI_189_DATA 0x0000032B
+#define DDRSS_PI_190_DATA 0x00001035
+#define DDRSS_PI_191_DATA 0x0000032B
+#define DDRSS_PI_192_DATA 0x04001035
 #define DDRSS_PI_193_DATA 0x01010404
 #define DDRSS_PI_194_DATA 0x00001501
-#define DDRSS_PI_195_DATA 0x00270027
+#define DDRSS_PI_195_DATA 0x002A002A
 #define DDRSS_PI_196_DATA 0x01000100
 #define DDRSS_PI_197_DATA 0x00000100
 #define DDRSS_PI_198_DATA 0x00000000
-#define DDRSS_PI_199_DATA 0x05090903
+#define DDRSS_PI_199_DATA 0x05080803
 #define DDRSS_PI_200_DATA 0x01011B1B
 #define DDRSS_PI_201_DATA 0x01010101
 #define DDRSS_PI_202_DATA 0x000C0C0A
 #define DDRSS_PI_203_DATA 0x00000000
 #define DDRSS_PI_204_DATA 0x00000000
 #define DDRSS_PI_205_DATA 0x04000000
-#define DDRSS_PI_206_DATA 0x0C021212
+#define DDRSS_PI_206_DATA 0x0C021111
 #define DDRSS_PI_207_DATA 0x0404020C
 #define DDRSS_PI_208_DATA 0x00090031
-#define DDRSS_PI_209_DATA 0x001B0043
-#define DDRSS_PI_210_DATA 0x001B0043
+#define DDRSS_PI_209_DATA 0x001E0046
+#define DDRSS_PI_210_DATA 0x001E0046
 #define DDRSS_PI_211_DATA 0x01010101
-#define DDRSS_PI_212_DATA 0x0003000D
-#define DDRSS_PI_213_DATA 0x000301D3
-#define DDRSS_PI_214_DATA 0x010001D3
+#define DDRSS_PI_212_DATA 0x0004000D
+#define DDRSS_PI_213_DATA 0x00040216
+#define DDRSS_PI_214_DATA 0x01000216
 #define DDRSS_PI_215_DATA 0x000E000E
-#define DDRSS_PI_216_DATA 0x01D40100
-#define DDRSS_PI_217_DATA 0x010001D4
-#define DDRSS_PI_218_DATA 0x01D401D4
+#define DDRSS_PI_216_DATA 0x02170100
+#define DDRSS_PI_217_DATA 0x01000217
+#define DDRSS_PI_218_DATA 0x02170217
 #define DDRSS_PI_219_DATA 0x301B3200
 #define DDRSS_PI_220_DATA 0x0101301B
 #define DDRSS_PI_221_DATA 0x0A070601
-#define DDRSS_PI_222_DATA 0x1C11090D
-#define DDRSS_PI_223_DATA 0x1C110913
-#define DDRSS_PI_224_DATA 0x000C0013
+#define DDRSS_PI_222_DATA 0x1F130A0D
+#define DDRSS_PI_223_DATA 0x1F130A14
+#define DDRSS_PI_224_DATA 0x000C0014
 #define DDRSS_PI_225_DATA 0x00001000
 #define DDRSS_PI_226_DATA 0x00000C00
 #define DDRSS_PI_227_DATA 0x00001000
 #define DDRSS_PI_228_DATA 0x00000C00
 #define DDRSS_PI_229_DATA 0x02001000
-#define DDRSS_PI_230_DATA 0x0021000D
-#define DDRSS_PI_231_DATA 0x002101D3
-#define DDRSS_PI_232_DATA 0x000001D3
+#define DDRSS_PI_230_DATA 0x0024000D
+#define DDRSS_PI_231_DATA 0x00240216
+#define DDRSS_PI_232_DATA 0x00000216
 #define DDRSS_PI_233_DATA 0x00001900
 #define DDRSS_PI_234_DATA 0x32000056
 #define DDRSS_PI_235_DATA 0x06000101
-#define DDRSS_PI_236_DATA 0x00300204
-#define DDRSS_PI_237_DATA 0x3212005A
+#define DDRSS_PI_236_DATA 0x00330204
+#define DDRSS_PI_237_DATA 0x32120059
 #define DDRSS_PI_238_DATA 0x17000301
-#define DDRSS_PI_239_DATA 0x00300C12
-#define DDRSS_PI_240_DATA 0x3212005A
+#define DDRSS_PI_239_DATA 0x00330C11
+#define DDRSS_PI_240_DATA 0x32120059
 #define DDRSS_PI_241_DATA 0x17000301
-#define DDRSS_PI_242_DATA 0x00000C12
+#define DDRSS_PI_242_DATA 0x00000C11
 #define DDRSS_PI_243_DATA 0x05040900
 #define DDRSS_PI_244_DATA 0x00040900
 #define DDRSS_PI_245_DATA 0x00000315
 #define DDRSS_PI_246_DATA 0x20010004
 #define DDRSS_PI_247_DATA 0x0A0A0A03
-#define DDRSS_PI_248_DATA 0x2B0F0000
-#define DDRSS_PI_249_DATA 0x24140026
-#define DDRSS_PI_250_DATA 0x0000731B
-#define DDRSS_PI_251_DATA 0x20070054
-#define DDRSS_PI_252_DATA 0x1B131B1C
-#define DDRSS_PI_253_DATA 0x2B0F0000
-#define DDRSS_PI_254_DATA 0x24140026
-#define DDRSS_PI_255_DATA 0x0000731B
-#define DDRSS_PI_256_DATA 0x20070054
-#define DDRSS_PI_257_DATA 0x1B131B1C
+#define DDRSS_PI_248_DATA 0x31110000
+#define DDRSS_PI_249_DATA 0x2917002C
+#define DDRSS_PI_250_DATA 0x0000838E
+#define DDRSS_PI_251_DATA 0x20080060
+#define DDRSS_PI_252_DATA 0x1E161E20
+#define DDRSS_PI_253_DATA 0x31110000
+#define DDRSS_PI_254_DATA 0x2917002C
+#define DDRSS_PI_255_DATA 0x0000838E
+#define DDRSS_PI_256_DATA 0x20080060
+#define DDRSS_PI_257_DATA 0x1E161E20
 #define DDRSS_PI_258_DATA 0x00000000
 #define DDRSS_PI_259_DATA 0x000000B2
 #define DDRSS_PI_260_DATA 0x000006F4
-#define DDRSS_PI_261_DATA 0x00001C5C
-#define DDRSS_PI_262_DATA 0x00011B98
-#define DDRSS_PI_263_DATA 0x00001C5C
-#define DDRSS_PI_264_DATA 0x00011B98
-#define DDRSS_PI_265_DATA 0x02D30014
-#define DDRSS_PI_266_DATA 0x030302D3
+#define DDRSS_PI_261_DATA 0x0000206A
+#define DDRSS_PI_262_DATA 0x00014424
+#define DDRSS_PI_263_DATA 0x0000206A
+#define DDRSS_PI_264_DATA 0x00014424
+#define DDRSS_PI_265_DATA 0x033B0014
+#define DDRSS_PI_266_DATA 0x0303033B
 #define DDRSS_PI_267_DATA 0x00000003
 #define DDRSS_PI_268_DATA 0x00000000
-#define DDRSS_PI_269_DATA 0x0A040503
-#define DDRSS_PI_270_DATA 0x00000A04
+#define DDRSS_PI_269_DATA 0x0B040503
+#define DDRSS_PI_270_DATA 0x00000B04
 #define DDRSS_PI_271_DATA 0x00002710
 #define DDRSS_PI_272_DATA 0x000186A0
 #define DDRSS_PI_273_DATA 0x00000005
 #define DDRSS_PI_274_DATA 0x00000064
 #define DDRSS_PI_275_DATA 0x00000014
-#define DDRSS_PI_276_DATA 0x0005B18F
+#define DDRSS_PI_276_DATA 0x000681C8
 #define DDRSS_PI_277_DATA 0x000186A0
 #define DDRSS_PI_278_DATA 0x00000005
-#define DDRSS_PI_279_DATA 0x00000E94
-#define DDRSS_PI_280_DATA 0x000002D3
-#define DDRSS_PI_281_DATA 0x0005B18F
+#define DDRSS_PI_279_DATA 0x000010A9
+#define DDRSS_PI_280_DATA 0x0000033B
+#define DDRSS_PI_281_DATA 0x000681C8
 #define DDRSS_PI_282_DATA 0x000186A0
 #define DDRSS_PI_283_DATA 0x00000005
-#define DDRSS_PI_284_DATA 0x00000E94
-#define DDRSS_PI_285_DATA 0x010002D3
+#define DDRSS_PI_284_DATA 0x000010A9
+#define DDRSS_PI_285_DATA 0x0100033B
 #define DDRSS_PI_286_DATA 0x00320040
 #define DDRSS_PI_287_DATA 0x00010008
-#define DDRSS_PI_288_DATA 0x074A0040
-#define DDRSS_PI_289_DATA 0x00010038
-#define DDRSS_PI_290_DATA 0x074A0040
-#define DDRSS_PI_291_DATA 0x00000338
-#define DDRSS_PI_292_DATA 0x0028005D
+#define DDRSS_PI_288_DATA 0x08550040
+#define DDRSS_PI_289_DATA 0x00010040
+#define DDRSS_PI_290_DATA 0x08550040
+#define DDRSS_PI_291_DATA 0x00000340
+#define DDRSS_PI_292_DATA 0x0028006A
 #define DDRSS_PI_293_DATA 0x03040404
 #define DDRSS_PI_294_DATA 0x00000303
 #define DDRSS_PI_295_DATA 0x01010000
@@ -967,29 +967,29 @@
 #define DDRSS_PHY_92_DATA 0x02000010
 #define DDRSS_PHY_93_DATA 0x00000005
 #define DDRSS_PHY_94_DATA 0x51516042
-#define DDRSS_PHY_95_DATA 0x31C06000
+#define DDRSS_PHY_95_DATA 0x31C06001
 #define DDRSS_PHY_96_DATA 0x07AB0340
 #define DDRSS_PHY_97_DATA 0x00C0C001
-#define DDRSS_PHY_98_DATA 0x0D000000
-#define DDRSS_PHY_99_DATA 0x000D0C0C
+#define DDRSS_PHY_98_DATA 0x0E000000
+#define DDRSS_PHY_99_DATA 0x000E0D0D
 #define DDRSS_PHY_100_DATA 0x42100010
-#define DDRSS_PHY_101_DATA 0x010C073E
-#define DDRSS_PHY_102_DATA 0x000F0C32
+#define DDRSS_PHY_101_DATA 0x010C083E
+#define DDRSS_PHY_102_DATA 0x000F0C37
 #define DDRSS_PHY_103_DATA 0x01000140
-#define DDRSS_PHY_104_DATA 0x011E0120
+#define DDRSS_PHY_104_DATA 0x01470120
 #define DDRSS_PHY_105_DATA 0x00000C00
-#define DDRSS_PHY_106_DATA 0x000002DD
+#define DDRSS_PHY_106_DATA 0x00000322
 #define DDRSS_PHY_107_DATA 0x00030200
 #define DDRSS_PHY_108_DATA 0x02800000
 #define DDRSS_PHY_109_DATA 0x80800000
-#define DDRSS_PHY_110_DATA 0x000D2010
+#define DDRSS_PHY_110_DATA 0x000E2010
 #define DDRSS_PHY_111_DATA 0x76543210
 #define DDRSS_PHY_112_DATA 0x00000008
-#define DDRSS_PHY_113_DATA 0x045D045D
-#define DDRSS_PHY_114_DATA 0x045D045D
-#define DDRSS_PHY_115_DATA 0x045D045D
-#define DDRSS_PHY_116_DATA 0x045D045D
-#define DDRSS_PHY_117_DATA 0x0000045D
+#define DDRSS_PHY_113_DATA 0x04A104A1
+#define DDRSS_PHY_114_DATA 0x04A104A1
+#define DDRSS_PHY_115_DATA 0x04A104A1
+#define DDRSS_PHY_116_DATA 0x04A104A1
+#define DDRSS_PHY_117_DATA 0x000004A1
 #define DDRSS_PHY_118_DATA 0x0000A000
 #define DDRSS_PHY_119_DATA 0x00A000A0
 #define DDRSS_PHY_120_DATA 0x00A000A0
@@ -999,7 +999,7 @@
 #define DDRSS_PHY_124_DATA 0x00A000A0
 #define DDRSS_PHY_125_DATA 0x00A000A0
 #define DDRSS_PHY_126_DATA 0x00A000A0
-#define DDRSS_PHY_127_DATA 0x00B200A0
+#define DDRSS_PHY_127_DATA 0x017F00A0
 #define DDRSS_PHY_128_DATA 0x01000000
 #define DDRSS_PHY_129_DATA 0x00000000
 #define DDRSS_PHY_130_DATA 0x00000000
@@ -1223,29 +1223,29 @@
 #define DDRSS_PHY_348_DATA 0x02000010
 #define DDRSS_PHY_349_DATA 0x00000005
 #define DDRSS_PHY_350_DATA 0x51516042
-#define DDRSS_PHY_351_DATA 0x31C06000
+#define DDRSS_PHY_351_DATA 0x31C06001
 #define DDRSS_PHY_352_DATA 0x07AB0340
 #define DDRSS_PHY_353_DATA 0x00C0C001
-#define DDRSS_PHY_354_DATA 0x0D000000
-#define DDRSS_PHY_355_DATA 0x000D0C0C
+#define DDRSS_PHY_354_DATA 0x0E000000
+#define DDRSS_PHY_355_DATA 0x000E0D0D
 #define DDRSS_PHY_356_DATA 0x42100010
-#define DDRSS_PHY_357_DATA 0x010C073E
-#define DDRSS_PHY_358_DATA 0x000F0C32
+#define DDRSS_PHY_357_DATA 0x010C083E
+#define DDRSS_PHY_358_DATA 0x000F0C37
 #define DDRSS_PHY_359_DATA 0x01000140
-#define DDRSS_PHY_360_DATA 0x011E0120
+#define DDRSS_PHY_360_DATA 0x01470120
 #define DDRSS_PHY_361_DATA 0x00000C00
-#define DDRSS_PHY_362_DATA 0x000002DD
+#define DDRSS_PHY_362_DATA 0x00000322
 #define DDRSS_PHY_363_DATA 0x00030200
 #define DDRSS_PHY_364_DATA 0x02800000
 #define DDRSS_PHY_365_DATA 0x80800000
-#define DDRSS_PHY_366_DATA 0x000D2010
+#define DDRSS_PHY_366_DATA 0x000E2010
 #define DDRSS_PHY_367_DATA 0x76543210
 #define DDRSS_PHY_368_DATA 0x00000008
-#define DDRSS_PHY_369_DATA 0x045D045D
-#define DDRSS_PHY_370_DATA 0x045D045D
-#define DDRSS_PHY_371_DATA 0x045D045D
-#define DDRSS_PHY_372_DATA 0x045D045D
-#define DDRSS_PHY_373_DATA 0x0000045D
+#define DDRSS_PHY_369_DATA 0x04A104A1
+#define DDRSS_PHY_370_DATA 0x04A104A1
+#define DDRSS_PHY_371_DATA 0x04A104A1
+#define DDRSS_PHY_372_DATA 0x04A104A1
+#define DDRSS_PHY_373_DATA 0x000004A1
 #define DDRSS_PHY_374_DATA 0x0000A000
 #define DDRSS_PHY_375_DATA 0x00A000A0
 #define DDRSS_PHY_376_DATA 0x00A000A0
@@ -1255,7 +1255,7 @@
 #define DDRSS_PHY_380_DATA 0x00A000A0
 #define DDRSS_PHY_381_DATA 0x00A000A0
 #define DDRSS_PHY_382_DATA 0x00A000A0
-#define DDRSS_PHY_383_DATA 0x00B200A0
+#define DDRSS_PHY_383_DATA 0x017F00A0
 #define DDRSS_PHY_384_DATA 0x01000000
 #define DDRSS_PHY_385_DATA 0x00000000
 #define DDRSS_PHY_386_DATA 0x00000000
@@ -1479,29 +1479,29 @@
 #define DDRSS_PHY_604_DATA 0x02000010
 #define DDRSS_PHY_605_DATA 0x00000005
 #define DDRSS_PHY_606_DATA 0x51516042
-#define DDRSS_PHY_607_DATA 0x31C06000
+#define DDRSS_PHY_607_DATA 0x31C06001
 #define DDRSS_PHY_608_DATA 0x07AB0340
 #define DDRSS_PHY_609_DATA 0x00C0C001
-#define DDRSS_PHY_610_DATA 0x0D000000
-#define DDRSS_PHY_611_DATA 0x000D0C0C
+#define DDRSS_PHY_610_DATA 0x0E000000
+#define DDRSS_PHY_611_DATA 0x000E0D0D
 #define DDRSS_PHY_612_DATA 0x42100010
-#define DDRSS_PHY_613_DATA 0x010C073E
-#define DDRSS_PHY_614_DATA 0x000F0C32
+#define DDRSS_PHY_613_DATA 0x010C083E
+#define DDRSS_PHY_614_DATA 0x000F0C37
 #define DDRSS_PHY_615_DATA 0x01000140
-#define DDRSS_PHY_616_DATA 0x011E0120
+#define DDRSS_PHY_616_DATA 0x01470120
 #define DDRSS_PHY_617_DATA 0x00000C00
-#define DDRSS_PHY_618_DATA 0x000002DD
+#define DDRSS_PHY_618_DATA 0x00000322
 #define DDRSS_PHY_619_DATA 0x00030200
 #define DDRSS_PHY_620_DATA 0x02800000
 #define DDRSS_PHY_621_DATA 0x80800000
-#define DDRSS_PHY_622_DATA 0x000D2010
+#define DDRSS_PHY_622_DATA 0x000E2010
 #define DDRSS_PHY_623_DATA 0x76543210
 #define DDRSS_PHY_624_DATA 0x00000008
-#define DDRSS_PHY_625_DATA 0x045D045D
-#define DDRSS_PHY_626_DATA 0x045D045D
-#define DDRSS_PHY_627_DATA 0x045D045D
-#define DDRSS_PHY_628_DATA 0x045D045D
-#define DDRSS_PHY_629_DATA 0x0000045D
+#define DDRSS_PHY_625_DATA 0x04A104A1
+#define DDRSS_PHY_626_DATA 0x04A104A1
+#define DDRSS_PHY_627_DATA 0x04A104A1
+#define DDRSS_PHY_628_DATA 0x04A104A1
+#define DDRSS_PHY_629_DATA 0x000004A1
 #define DDRSS_PHY_630_DATA 0x0000A000
 #define DDRSS_PHY_631_DATA 0x00A000A0
 #define DDRSS_PHY_632_DATA 0x00A000A0
@@ -1511,7 +1511,7 @@
 #define DDRSS_PHY_636_DATA 0x00A000A0
 #define DDRSS_PHY_637_DATA 0x00A000A0
 #define DDRSS_PHY_638_DATA 0x00A000A0
-#define DDRSS_PHY_639_DATA 0x00B200A0
+#define DDRSS_PHY_639_DATA 0x017F00A0
 #define DDRSS_PHY_640_DATA 0x01000000
 #define DDRSS_PHY_641_DATA 0x00000000
 #define DDRSS_PHY_642_DATA 0x00000000
@@ -1735,29 +1735,29 @@
 #define DDRSS_PHY_860_DATA 0x02000010
 #define DDRSS_PHY_861_DATA 0x00000005
 #define DDRSS_PHY_862_DATA 0x51516042
-#define DDRSS_PHY_863_DATA 0x31C06000
+#define DDRSS_PHY_863_DATA 0x31C06001
 #define DDRSS_PHY_864_DATA 0x07AB0340
 #define DDRSS_PHY_865_DATA 0x00C0C001
-#define DDRSS_PHY_866_DATA 0x0D000000
-#define DDRSS_PHY_867_DATA 0x000D0C0C
+#define DDRSS_PHY_866_DATA 0x0E000000
+#define DDRSS_PHY_867_DATA 0x000E0D0D
 #define DDRSS_PHY_868_DATA 0x42100010
-#define DDRSS_PHY_869_DATA 0x010C073E
-#define DDRSS_PHY_870_DATA 0x000F0C32
+#define DDRSS_PHY_869_DATA 0x010C083E
+#define DDRSS_PHY_870_DATA 0x000F0C37
 #define DDRSS_PHY_871_DATA 0x01000140
-#define DDRSS_PHY_872_DATA 0x011E0120
+#define DDRSS_PHY_872_DATA 0x01470120
 #define DDRSS_PHY_873_DATA 0x00000C00
-#define DDRSS_PHY_874_DATA 0x000002DD
+#define DDRSS_PHY_874_DATA 0x00000322
 #define DDRSS_PHY_875_DATA 0x00030200
 #define DDRSS_PHY_876_DATA 0x02800000
 #define DDRSS_PHY_877_DATA 0x80800000
-#define DDRSS_PHY_878_DATA 0x000D2010
+#define DDRSS_PHY_878_DATA 0x000E2010
 #define DDRSS_PHY_879_DATA 0x76543210
 #define DDRSS_PHY_880_DATA 0x00000008
-#define DDRSS_PHY_881_DATA 0x045D045D
-#define DDRSS_PHY_882_DATA 0x045D045D
-#define DDRSS_PHY_883_DATA 0x045D045D
-#define DDRSS_PHY_884_DATA 0x045D045D
-#define DDRSS_PHY_885_DATA 0x0000045D
+#define DDRSS_PHY_881_DATA 0x04A104A1
+#define DDRSS_PHY_882_DATA 0x04A104A1
+#define DDRSS_PHY_883_DATA 0x04A104A1
+#define DDRSS_PHY_884_DATA 0x04A104A1
+#define DDRSS_PHY_885_DATA 0x000004A1
 #define DDRSS_PHY_886_DATA 0x0000A000
 #define DDRSS_PHY_887_DATA 0x00A000A0
 #define DDRSS_PHY_888_DATA 0x00A000A0
@@ -1767,7 +1767,7 @@
 #define DDRSS_PHY_892_DATA 0x00A000A0
 #define DDRSS_PHY_893_DATA 0x00A000A0
 #define DDRSS_PHY_894_DATA 0x00A000A0
-#define DDRSS_PHY_895_DATA 0x00B200A0
+#define DDRSS_PHY_895_DATA 0x017F00A0
 #define DDRSS_PHY_896_DATA 0x01000000
 #define DDRSS_PHY_897_DATA 0x00000000
 #define DDRSS_PHY_898_DATA 0x00000000
diff --git a/arch/arm/dts/k3-am6754-r5-phycore-som-8gb.dts b/arch/arm/dts/k3-am6754-r5-phycore-som-8gb.dts
index 56367791996..c76e6091da2 100644
--- a/arch/arm/dts/k3-am6754-r5-phycore-som-8gb.dts
+++ b/arch/arm/dts/k3-am6754-r5-phycore-som-8gb.dts
@@ -8,7 +8,7 @@
 #include "k3-am6754-phyboard-rigel.dts"
 #include "k3-am6754-phyboard-rigel-u-boot.dtsi"
 
-#include "k3-am67-phycore-som-lpddr4-4gb.dtsi"
+#include "k3-am67-phycore-som-lpddr4-8gb.dtsi"
 #include "k3-am62a-ddr.dtsi"
 
 / {
-- 
2.43.0


